lms/services/blackboard_api/_schemas.py

```diff
@@ -0,0 +1,26 @@
+"""
+Schemas for Blackboard API responses.
+
+See: https://developer.blackboard.com/portal/displayApi
+"""
+from marshmallow import EXCLUDE, Schema, fields, post_load
+
+from lms.validation._base import RequestsResponseSchema
+
+
+class BlackboardListFilesSchema(RequestsResponseSchema):
+    """Schema for Blackboard API /courses/{courseId}/resources responses."""
+
+    class ResultsSchema(Schema):
```

Comment:
I know this is inside `BlackboardListFilesSchema` so it's kinda namespaced already but I think `FileSchema` might be a better name .

Reply:
Done. Added a small docstring too